lib/eslint.js

```diff
@@ -27,7 +27,8 @@ var lodash = require("lodash"),
     validator = require("./config/config-validator"),
     replacements = require("../conf/replacements.json"),
     assert = require("assert"),
-    CodePathAnalyzer = require("./code-path-analysis/code-path-analyzer");
+    CodePathAnalyzer = require("./code-path-analysis/code-path-analyzer"),
+    ESLINT_VERSION = require("../package.json").version;
```

Comment:
This is [exactly how we do it](https://github.com/eslint/eslint/blob/7051465f9090959da031b38036673106310bf1be/lib/cli.js#L144) for the `--version` CLI option.